cookbooks/aws-parallelcluster-environment/resources/lustre/lustre_centos7.rb

```diff
@@ -104,6 +104,13 @@ def find_centos_minor_version
   os_minor_version
 end
 
+def find_kernel_patch_version
+  # kernel release is in the form 3.10.0-1127.8.2.el7.x86_64
+  kernel_patch_version = node['cluster']['kernel_release'].match(/^\d+\.\d+\.\d+-(\d+)\..*$/)
+  raise "Unable to retrieve the kernel patch version from #{node['cluster']['kernel_release']}." unless kernel_patch_version
+  kernel_patch_version[1]
+end
+
```

Comment:
Can we have a unit test for this function?

Reply:
Added